en/docs/devops-and-ci-cd/manage-configuration-groups.md

```diff
@@ -0,0 +1,112 @@
+
+# Manage Configuration Groups
+
+**Configuration groups** are collections of key-value pairs that can be linked to any component within the organization, enabling centralized configuration management.
```

Comment:
Suggestion:
Choreo allows you to create configuration groups to easily manage reusable configurations across components within your organization. A configuration group is a collection of key-value pairs that centralizes configuration management and ensures consistent configuration application across environments. 
Configuration values can be defined per environment. During deployment, configuration groups inject keys as environment variables or file mounts. Choreo manages and applies these configurations during deployment and promotion.